Source/WebCore/platform/graphics/filters/FilterFunction.h

```diff
@@ -93,7 +94,7 @@ class FilterFunction : public RefCounted<FilterFunction> {
     static AtomString sourceGraphicName() { return filterName(Type::SourceGraphic); }
     AtomString filterName() const { return filterName(m_filterType); }
 
-    virtual bool supportsAcceleratedRendering() const { return false; }
+    virtual OptionSet<FilterRenderingMode> supportedFilterRenderingModes() const { return { FilterRenderingMode::Software }; }
```

Comment:
Nit: I think the braces around `FilterRenderingMode::Software` aren't needed, since `OptionSet` has an implicit constructor that takes the enum type.